infra/README.md

```diff
@@ -1,23 +1,23 @@
-# A cli tool to create/scale/delete k8s clusters and deploy manifest files.
+# infra: A cli tool to create/scale/delete k8s clusters and deploy manifest files.
 Currently it supports GKE, but it is designed in a way that adding more providers should be easy.
 
 ### Parsing of files
-Files passed to Prombench will be parsed using golang templates, to skip parsing and load the file as is, use `noparse` suffix.
+Files passed to `infra` will be parsed using golang templates, to skip parsing and load the file as is, use `noparse` suffix.
 
 Eg. `somefile.yaml` will be parsed, whereas `somefile_noparse.yaml` will not be parsed.
 
 ### Example for building the docker image
 From the repository root:
 ```
-$ make docker DOCKERFILE_PATH=prombench/cmd/prombench/Dockerfile DOCKER_IMAGE_NAME=prombench DOCKER_IMAGE_TAG=2.0.2
+$ make docker DOCKERFILE_PATH=infra/Dockerfile DOCKER_IMAGE_NAME=infra DOCKER_IMAGE_TAG=2.0.2
```

Comment:
I would say lets move pkg inside the infra folder and simplify this docker build.

Than can completely remove this paragraph as the docker build would become just `docker build .`

Reply:
other tools like scalar are using `pkg` so I think it's fine it to stay in the repository root. I've opened a PR to add dockerbuild context to the circleci orb (https://github.com/prometheus/circleci/pull/5) so that we can do build images without having the root repository context. Once that's done, I we'll be able to replace all docker instructions(in READMEs) in the `test-infra` repo with just `docker build .` and the circleci config can use the `dockerbuild_context`.